### ISSUE TITLE
Instead of showing Score, Upvote, and Downvote, show only Score and Upvote Percentage (or sentiment icon)

### ISSUE THREAD

**Issue by dessalines** [dessalines/jerboa#1372]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

@MV-GH I'd like your feedback before I start working on this.

Currently, on every post / comment, we show 3 pieces of info:

- A score in the header bar
- Upvote and downvote count in the footer / action bar.

Sample:

![Screenshot_2024-02-12-12-52-32-271-edit_com jerboa_1](https://github.com/dessalines/jerboa/assets/930215/c6793f47-0a47-4a95-8421-9002a2a3116e)

It'd be nice to simplify this (like some other apps I've seen), and show only

- A score and an upvote percentage in the header bar.
  - We could also hide the upvote percentage if its above a certain threshold (IE 80% liked)
  - And instead of a percentage number, we could show one of these [sentiment icons](https://fonts.google.com/icons?icon.query=sentiment)

This would move the pieces of info shown from 3 down to 2 (or 1 if its a well liked item).

**Comment by MV-GH** [comment]
I am open for this, but I would lean on making this a setting at first. Also I prefer percentage (Hide this if its 95%+)

**Comment by dessalines** [comment]
I've decided that this is a pervasive enough setting that it should probably live in the back-end.

I can implement this *before* we add that back-end setting referenced above, by using a dummy setting.

**Comment by MV-GH** [comment]
We are gonna need something dummy regardless for backwards compatibility :/

Lemmy.world is not even on 0.19 yet so the backwards compatibility is still pretty important.

or for Lemmy version which don't implement that feature yet, we can just assume they want the old behaviour and only make the setting available for instances that support the feature

**Comment by dessalines** [comment]
For sure, mainly I just want to hold off on adding a DB migration in jerboa, and instead make this a back-end user setting.

**Comment by MV-GH** [comment]
Isn't this then a setting on Acccount too? like listing etc

**Comment by dessalines** [comment]
Yep, I imagine it would be good to persist there also, just like `default_post_listing_type`, etc.

**Comment by MV-GH** [comment]
Since which lemmy version was postlistingMode added?
